src/sentry/buffer/redis.py

```diff
@@ -54,10 +55,11 @@ def flush(self):
 class RedisBuffer(Buffer):
     key_expire = 60 * 60  # 1 hour
     pending_key = 'b:p'
-    incr_batch_size = 2
 
-    def __init__(self, **options):
+    def __init__(self, pending_shards=1, incr_batch_size=2, **options):
         self.cluster, options = get_cluster_from_options('SENTRY_BUFFER_OPTIONS', options)
+        self.pending_shards = pending_shards
+        self.incr_batch_size = incr_batch_size
```

Comment:
What motivated this move from a class attribute to an instance one?

Reply:
I just made it configurable through an option. I figured "why not" since I was adding `pending_shards`, might as well make `incr_batch_size` also configurable.

Reply:
Cool, just making sure I didn't miss any changes here. :)